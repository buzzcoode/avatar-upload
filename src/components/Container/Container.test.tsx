import React from 'react';
import { render } from '@testing-library/react';
import { Container } from '.';

it('renders correctly', async () => {
  const { queryByTestId } = render(<Container>content</Container>);
  expect(queryByTestId('styled-container')).toMatchSnapshot();
});

it('renders default snapshot', async () => {
  const { container } = render(<Container>content</Container>);
  expect(container).toMatchSnapshot();
});

it('renders snapshot with colors', async () => {
  const { container } = render(<Container bg="primary">content</Container>);
  expect(container).toMatchSnapshot()
});