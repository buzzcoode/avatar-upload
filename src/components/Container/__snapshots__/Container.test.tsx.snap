// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.emotion-0 {
  color: black;
  background-color: background;
  padding: 16px;
}

<div
  class="emotion-0"
  data-testid="styled-container"
>
  content
</div>
`;

exports[`renders default snapshot 1`] = `
.emotion-0 {
  color: black;
  background-color: background;
  padding: 16px;
}

<div>
  <div
    class="emotion-0"
    data-testid="styled-container"
  >
    content
  </div>
</div>
`;

exports[`renders snapshot with colors 1`] = `
.emotion-0 {
  color: black;
  background-color: primary;
  padding: 16px;
}

<div>
  <div
    class="emotion-0"
    data-testid="styled-container"
  >
    content
  </div>
</div>
`;
